tests/test_forms.py

```diff
@@ -243,3 +244,14 @@ class CustomModel(Base):
 async def test_model_form_include_pk() -> None:
     Form = await get_model_form(model=User, engine=engine, form_include_pk=True)
     assert "id" in Form()._fields
+
+
+async def test_form_time_converter() -> None:
+    class ExampleModel(Base):
+        __tablename__ = "example_model"
+        id = Column(Integer, primary_key=True)
+        start_time = Column(Time)
```

Comment:
You can add this new field to here, I think it won't need a separate test.

Reply:
I mean if you add this Time field to the `User` class in there, you can just assert it in `test_model_form` and no need for the new test.

Reply:
got it.